cloudnet-app-lib/cloudnet-logging/src/main/java/eu/cloudnetservice/cloudnet/v2/logging/color/AnsiColorReplacer.java

```diff
@@ -0,0 +1,70 @@
+package eu.cloudnetservice.cloudnet.v2.logging.color;
+
+import org.fusesource.jansi.Ansi;
+
+import java.util.regex.Pattern;
+
+public class AnsiColorReplacer {
+    private static class ReplacementSpecification {
+
+        private final Pattern pattern;
+        private final String replacement;
+
+        private ReplacementSpecification(final Pattern pattern, final String replacement) {
+            this.pattern = pattern;
+            this.replacement = replacement;
+        }
+    }
+
+    private static ReplacementSpecification compile(ChatColor color, String ansi) {
+        return new ReplacementSpecification(Pattern.compile("(?i)" + color.toString()), ansi);
+    }
+
+    private static final ReplacementSpecification[] REPLACEMENTS = new ReplacementSpecification[]
+        {
+            compile(ChatColor.BLACK, Ansi.ansi().a(Ansi.Attribute.RESET).fg(Ansi.Color.BLACK).boldOff().toString()),
```

Comment:
```suggestion
            compile(ChatColor.BLACK, Ansi.ansi().reset().fg(Ansi.Color.BLACK).boldOff().toString()),
```

This is shorter and does the same according to the documentation.